src/lib/utils.js

```diff
@@ -33,18 +32,20 @@ const generateWaitSet = (trial, waitTime) => {
   return [waitTrial, trial]
 }
 
-const keypressResponse = (info) => {
-  const data = {
-    key_press: info.key
-  }
+// As of jspsych 7, we instantiate jsPsych where needed insead of importing it globally.
+// The jsPsych instance passed in here should be the same one used for the running task.
+const startKeypressListener = (jsPsych) => {
+  const keypressResponse = (info) => {
+    const data = {
+      key_press: info.key
+    }
 
-  jsPsych.finishTrial(data)
-}
+    jsPsych.finishTrial(data)
+  }
 
-const startKeypressListener = () => {
   let keyboardListener = jsPsych.pluginAPI.getKeyboardResponse({
     callback_function: keypressResponse,
-    valid_responses: jsPsych.ALL_KEYS,
+    valid_responses: ['ALL_KEYS'],
```

Comment:
This is a `jspsych` 7 difference mentioned in the [migration guide](https://www.jspsych.org/7.0/support/migration-v7/): filter response keys by certain string names instead of explicit constants.